tests/epics/motion/test_motor.py

```diff
@@ -118,6 +119,29 @@ async def test_motor_moving_well_2(sim_motor: motor.Motor) -> None:
     done.assert_called_once_with(s)
 
 
+async def test_motor_move_timeout(sim_motor: motor.Motor):
+    def do_timeout(value, wait=False, timeout=None):
+        # Check we were given the right timeout of move_time + DEFAULT_TIMEOUT
+        assert timeout == 10.3
+        raise ValueError("I Timed Out")
```

Comment:
Nit: Can this be a `TmeoutError`? Makes more sense

Reply:
Just making it a non-obvious error to make sure it bubbles up correctly, could make a custom Exception subclass for this instead to make it extra obvious